CONTRIBUTING.md

```diff
@@ -42,21 +42,23 @@ When submitting a pull request, please make sure to:
 
 To set up a development environment for this project, you will need to:
 
-* Install Python 3.8 or higher and make sure java is installed on your system
+* Install Python 3.9 or higher and make sure java is installed on your system
```

Comment:
One of the dev packages required Python 3.9. Are we okay dropping 3.8 at least for development?

Reply:
yep, we can drop 3.8 across the board.